spec/controllers/hyrax/batch_edits_controller_spec.rb

```diff
@@ -93,11 +97,26 @@
     let(:curation_concern) { create(:work1, user: user) }
 
     context 'when user has edit access' do
-      it "deletes collections with and without works in it" do
-        controller.batch = [collection1.id, collection2.id]
+      it "deletes collections with and without works in it and an empty admin set." do
+        controller.batch = [collection1.id, collection2.id, empty_adminset.id]
+        delete :destroy_collection, params: { update_type: "delete_all" }
+        expect { Collection.find(collection1.id) }.to raise_error(Ldp::Gone)
```

Comment:
can you check `expect(collection1).not_to exist(collection1.id)` rather than trapping for errors?